pkg/virt-launcher/virtwrap/converter/network.go

```diff
@@ -83,7 +83,7 @@ func CreateDomainInterfaces(vmi *v1.VirtualMachineInstance, c *ConverterContext)
 			domainIface.Type = "ethernet"
 			if iface.BootOrder != nil {
 				domainIface.BootOrder = &api.BootOrder{Order: *iface.BootOrder}
-			} else if !isS390X(vmi.Spec.Architecture) {
+			} else if NewArchConverter(vmi.Spec.Architecture).isROMTuningSupported() {
```

Comment:
The function already has `ConverterContext` as argument, so it should be simplified to
```suggestion
			} else if c.isROMTuningSupported() {
```

Reply:
Well, the architecture set in the context can differ from the one specified in the vmi.Spec.
The first one is the host arch, while the second one is the guest arch.
Here looks like it cares about the vmi.Spec.Arch, so I need to re-intantiate a new arch converter

Reply:
Yes, I will discuss it in a separate context. Thanks